test/phpunit/SecurityTest.php

```diff
@@ -839,7 +840,15 @@ public function testGetURLContent()
 		$url = 'ftp://mydomain.com';
 		$tmp = getURLContent($url);
 		print __METHOD__." url=".$url."\n";
-		$this->assertRegExp("/not supported/", $tmp['curl_error_msg'], "Should disable ftp connection");	// Test error if return does not contains 'not supported'
```

Comment:
I think a better way to fix this is to make a
_$tmpvar = preg_match('/not supported/', ...)_
then
_this->assertTrue($tmpvar);_

We have so few use of assertRegExp that removing it completely seems better for future.

Reply:
That is not better IMHO as the message that is reported in case of failure does not report the regex it is trying to match, while using the phpunit methods for regex shows this.  So it's easier to debug.

Or the other method is to add a method in the Common*Test class that now exists and do these conversions there: implement assertRegex and call the appropriate parent method according to the version of PHPUNIT.
When I wrote this there was no Common*Test class.